build/images/tf_operator/Dockerfile

```diff
@@ -11,8 +11,8 @@ FROM registry.access.redhat.com/ubi8/ubi:latest
 # TODO(jlewi): We should probably change the directory to /opt/kubeflow.
 RUN mkdir -p /opt/kubeflow/samples
 
-COPY tf_smoke.py /opt/kubeflow/samples/
-RUN chmod a+x /opt/kubeflow/samples/*
+#COPY tf_smoke.py /opt/kubeflow/samples/
+#RUN chmod a+x /opt/kubeflow/samples/*
```

Comment:
Please add some comments to illustrate why we need it.

Reply:
@gaocegege , the build is successful locally after commenting out the above lines.

The error seen without commenting the lines : 
```
Step 7/11 : COPY tf_smoke.py /opt/kubeflow/samples/
COPY failed: stat /var/lib/docker/tmp/docker-builder306018796/tf_smoke.py: no such file or directory
```

Reply:
@hougangliu , had suggested me to comment out the above lines.